.github/workflows/publish.yml

```diff
@@ -53,7 +53,14 @@ jobs:
           tag: ${{ steps.package-version.outputs.version }}
           artifacts: ./*.vsix
 
-      - name: release to VSCode marketplace
+      - name: release prerelease to VSCode marketplace
```

Comment:
same here if else is supported better to merge into one.

Reply:
done